packages/interactivity/src/store.ts

```diff
@@ -147,7 +149,45 @@ const handlers = {
 			return ( ...args: unknown[] ) => {
 				setNamespace( ns );
 				try {
-					return result( ...args );
+					const scope = getScope();
+					const _this = {};
+					Object.defineProperty( _this, 'state', {
+						configurable: false,
+						enumerable: true,
+						get() {
+							setNamespace( ns );
+							setScope( scope );
+							const val = stores.get( ns );
```

Comment:
I think the store returns an object, so you still need to access the state:
```suggestion
							const val = stores.get( ns ).state;
```